app/models/host/managed.rb

```diff
@@ -912,22 +912,6 @@ def recreate_config(only = nil)
     result
   end
 
-  # converts a name into ip address using DNS.
-  # if we are managing DNS, we can query the correct DNS server
-  # otherwise, use normal systems dns settings to resolv
-  def to_ip_address(name_or_ip)
```

Comment:
Yeah - please do it with Foreman::Deprecation. It shouldn't take much time & we ensure nothing breaks for sure. Maybe some test in some plugin indirectly relies on this, we'd rather not break anything without warning.